lib/addons/xray/xray-ds.yaml

```diff
@@ -11,111 +11,72 @@
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
 
-apiVersion: rbac.authorization.k8s.io/v1beta1
-kind: ClusterRoleBinding
+# create amazon-cloudwatch namespace
+apiVersion: v1
+kind: Namespace
 metadata:
-  name: xray-daemon
+  name: amazon-cloudwatch
   labels:
-    app: xray-daemon
+    name: amazon-cloudwatch
+---
+# create role binding for XRay SDK to read config map
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Why do we need these roles and role binding if the SA is not even referenced in the ds deployment?

Reply:
Makes sense. Removed it. Pleae check now.